optimum/habana/transformers/training_args.py

```diff
@@ -123,7 +123,7 @@ class GaudiTrainingArguments(TrainingArguments):
     )
 
     use_lazy_mode: Optional[bool] = field(
-        default=False,
+        default=True,
```

Comment:
@libinta Should we wait for SynapseAI 1.13 to change the default to True?

Reply:
I think we can merge it